src/libraries/shims/ApiCompat.proj

```diff
@@ -95,7 +95,26 @@
     </Exec>
 
     <Error Condition="'$(ApiCompatExitCode)' != '0'" Text="ApiCompat failed comparing netstandard to $(ApiCompatTarget)" />
+ 
+    <PropertyGroup>
+      <PreviousNetCoreApp>netcoreapp3.1</PreviousNetCoreApp>
```

Comment:
Is there a better way to calculate this instead of having to manually update it?

Reply:
I don't think you can calculate it.  When building 5.0 could you calculate we previously shipped 3.1?  Why not 3.2, 4.0, 4.1, 4.9?  You could see what was supported by the SDK, but that seems indirect and fragile.  I feel much better to have this one piece of declarative input that will break when it needs to be updated.

Reply:
Yeah I thought about it, but asking in case you came up with a good idea. It is not the end of the world though, sounds good, to break whenever we move NetCoreAppCurrent up and have to update it.